src/client/app/components/about.ts

```diff
@@ -65,13 +100,34 @@ export class About implements AfterViewInit {
         this.showChange.emit(false);
     }
 
-    cache = `
-    ${Strings().aboutStorage}
-    ${storageSize(localStorage, `playground_${environment.current.host}_snippets`, Strings().aboutSnippets)}
-    ${storageSize(sessionStorage, 'playground_intellisense', Strings().aboutIntellisense)}
-    `;
-
     setLanguage(languageCode: string) {
         this.currentChosenLanguage = languageCode;
     }
+
+    async changeConfig(newConfig: string) {
```

Comment:
Do you want to show the dialog only on "okClicked", like I did with the languages?  That way, if the user goes through the dropdown with a keyboard, they don't get inundated with alerts?